schema/snapcraft.yaml

```diff
@@ -88,6 +88,12 @@ properties:
           stop-command: ["daemon"]
         additionalProperties: false
         properties:
+          desktop:
+            type: string
+            description: path to a desktop file representing the app, relative to the prime directory
+          icon:
```

Comment:
That last case is where I'd like to be smart, detect it is a path and apply the correct snapcraft magic foo of prefixing $SNAP in that path.